VideoAPI/VideoApi.Services/AudioPlatformService.cs

```diff
@@ -56,6 +62,11 @@ public async Task<WowzaGetApplicationsResponse> GetAllAudioApplicationsInfoAsync
             }
             catch (AudioPlatformException ex)
             {
+                if (ex.StatusCode == HttpStatusCode.NotFound)
```

Comment:
This code block should be replaced with a method to handle wowza errors

Reply:
Done